docs/references/room.md

```diff
@@ -2,624 +2,242 @@
-import {
-  Wechaty,
-  UrlLink,
-  MiniProgram,
-}  from 'wechaty'
-
-const bot = new Wechaty()
-await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'wechaty'})
-
-// 1. Send text inside Room
-
-await room.say('Hello world!')
-
-// 2. Send media file inside Room
+所有的微信群，都会用一个 Room 对象封装。
 
-import { FileBox }  from 'file-box'
-const fileBox1 = FileBox.fromUrl('https://wechaty.github.io/wechaty/images/bot-qr-code.png')
-const fileBox2 = FileBox.fromLocal('/tmp/text.txt')
-await room.say(fileBox1)
-await room.say(fileBox2)
+<a name="wechaty.user.room.Room.__init__"></a>
+#### \_\_init\_\_
 
-// 3. Send Contact Card in a room
-
-const contactCard = await bot.Contact.find({name: 'lijiarui'}) // change 'lijiarui' to any of the room member
-await room.say(contactCard)
-
-// 4. Send text inside room and mention @mention contact
-
-const members = await room.memberAll() // all members in this room
-const someMembers = members.slice(0, 3);
-await room.say('Hello world!', ...someMembers)
-
-// 5. send Link inside room
-
-const linkPayload = new UrlLink({
-  description : 'WeChat Bot SDK for Individual Account, Powered by TypeScript, Docker, and Love',
-  thumbnailUrl: 'https://avatars0.githubusercontent.com/u/25162437?s=200&v=4',
-  title       : 'Welcome to Wechaty',
-  url         : 'https://github.com/wechaty/wechaty',
-})
-await room.say(linkPayload)
-
-// 6. send MiniProgram (only supported by `wechaty-puppet-macpro`)
-
-const miniProgram = new MiniProgram ({
-  appid              : 'gh_0aa444a25adc',
-  title              : '我正在使用Authing认证身份，你也来试试吧',
-  pagePath           : 'routes/explore.html',
-  description        : '身份管家',
-  thumbUrl           : '30590201000452305002010002041092541302033d0af802040b30feb602045df0c2c5042b777875706c6f61645f31373533353339353230344063686174726f6f6d3131355f313537363035393538390204010400030201000400',
-  thumbKey           : '42f8609e62817ae45cf7d8fefb532e83',
-});
-await room.say(miniProgram);
+```python
+ | __init__(room_id: str) -> None
 ```
+构造函数。
 
-### room.on\(event, listener\) ⇒ `this`
+<a name="wechaty.user.room.Room.on"></a>
+#### on
 
-**Kind**: instance method of [`Room`](room.md#Room) **Returns**: `this` - - Room for chain
-
-| Param | Type | Description |
-| :--- | :--- | :--- |
-| event | [`RoomEventName`](room.md#RoomEventName) | Emit WechatyEvent |
-| listener | [`RoomEventFunction`](room.md#RoomEventFunction) | Depends on the WechatyEvent |
-
-#### Example _\(Event:join \)_
-
-```javascript
-const bot = new Wechaty()
-await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'topic of your room'}) // change `event-room` to any room topic in your wechat
-if (room) {
-  room.on('join', (room, inviteeList, inviter) => {
-    const nameList = inviteeList.map(c => c.name()).join(',')
-    console.log(`Room got new member ${nameList}, invited by ${inviter}`)
-  })
-}
+```python
+ | on(event_name: str, func)
 ```
+捕获事件。
```

Comment:
a ~ haha ~ , this description is so **concise**,  If you can give more detailed description, you will get into deep with `python-wechaty`.